include/fastrtps/qos/QosPolicies.h

```diff
@@ -62,7 +62,6 @@ constexpr DestinationOrderQosPolicyKind BY_RECEPTION_TIMESTAMP_DESTINATIONORDER_
 constexpr DestinationOrderQosPolicyKind BY_SOURCE_TIMESTAMP_DESTINATIONORDER_QOS =
         DestinationOrderQosPolicyKind::BY_SOURCE_TIMESTAMP_DESTINATIONORDER_QOS;
 using DestinationOrderQosPolicy = fastdds::dds::DestinationOrderQosPolicy;
-using UserDataQosPolicy = fastdds::dds::UserDataQosPolicy;
```

Comment:
Why treat UserDataQosPolicy different from the rest of QosPolicy classes?

Reply:
The linker complained on non-defined `UserDataQosPolicy::addToCDRMessage`. This class is only being used inside other classes (ProxyData and DDS QoS), so it is almost impossible for a user to have been using it. In fact, as part of the DDS PSM changes we are making, this has already been done.